src/GoTrueApi.ts

```diff
@@ -697,9 +697,9 @@ export default class GoTrueApi {
     }
     const { event, session } = req.body
 
-    if (!event) throw new Error('Auth event missing!')
+    if (!event) throw new AuthEventMissingError()
```

Comment:
More a question than a review comment: when will we have auth events go missing? Not too sure how the events portion works.

If there's time, can I trouble you for a short overview pre/post sync?